common/eth2/src/types.rs

```diff
@@ -648,6 +648,12 @@ pub struct ValidatorBalancesQuery {
     pub id: Option<Vec<ValidatorId>>,
 }
 
+#[derive(Clone, Deserialize)]
```

Comment:
Might be worth adding a `deny_unknown_fields` attribute here for consistency with other endpoints and return error if user provided some unknown parameters.

```suggestion
#[derive(Clone, Deserialize)]
#[serde(deny_unknown_fields)]
```